### PR TITLE
[BOT-265] Action endpoint

### PR DESCRIPTION
**Pull request by blakeandrewwood**

### CODE REVIEW

File: src/Shell.tsx [hunk 1]
```diff
@@ -192,13 +213,14 @@ class ShellContainer extends React.Component<Props> implements ShellFunctions {
...
         // passed down to ShellContainer
+        connectionStatus: state.connection.connectionStatus,
         format: state.format,
         inputText: state.shell.input,
         showUploadButton: state.format.showUploadButton,
         strings: state.format.strings,
         // only used to create helper functions below
         locale: state.format.locale,
-        user: state.connection.user,
+        //user: state.connection.user,
```
Comment (blakeandrewwood) [inline src/Shell.tsx]: Remove commented out code

File: src/Shell.tsx [hunk 2]
```diff
@@ -220,7 +251,9 @@ export const Shell = connect(
         onChangeText: dispatchProps.onChangeText,
         // helper functions
         doCardAction: doCardAction(stateProps.botConnection, stateProps.user, stateProps.format.locale, dispatchProps.sendMessage, dispatchProps.addMessage),
-        sendMessage: (text: string) => dispatchProps.sendMessage(text, stateProps.user, stateProps.locale),
+        //sendMessage: (text: string, from: User, locale: string) => dispatchProps.sendMessage(text, stateProps.user, stateProps.locale),
```
Comment (blakeandrewwood) [inline src/Shell.tsx]: Remove commented out code

File: src/Chat.tsx [hunk 3]
```diff
@@ -183,71 +169,43 @@ export class Chat extends React.Component<ChatProps, {}> {
...
+        else {
+            axios.request<Activity[]>({
+                method: 'GET',
+                url: this.props.actionEndpointUrl,
+            })
+            .then((response) => {
+                const { data } = response
+                data.forEach((activity) => { 
+                    this.store.dispatch<ChatActions>({ activity, type: "Receive_Message"})
+                });
```
Comment (odannyc) [inline src/Chat.tsx]: If the original action endpoint errors out the bot won't say anything. Can we fallback to sending out the message to directline if we don't get a `200` back from the action endpoint

Comment (blakeandrewwood) [reply src/Chat.tsx]: We could, but would there ever be a situation where the action endpoint is not working, but directline is?

Comment (odannyc) [reply src/Chat.tsx]: Yes. Unfortunately the action endpoint is not bulletproof so if there's an error or something else goes wrong with that endpoint we can atleast fallback and try directline

Comment (odannyc) [reply src/Chat.tsx]: Just leaving this note here....
It was decided that we'll keep an eye on this to see if it happens enough.

**Pull request merged**